docs/develop/enable-nested-app-authentication-in-your-add-in.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Enable SSO in an Office Add-in using nested app authentication
+description: Learn how to enable SSO in an Office Add-in using nested app authentication.
+ms.date: 03/20/2024
+ms.topic: how-to
+ms.localizationpriority: medium
+
+---
+
+# Enable SSO in an Office Add-in using nested app authentication (preview)
+
+You can use the MSAL.js library (version 3.10 and later) with nested app authentication to use SSO from your Office Add-in. Using nested app authentication offers several advantages over the On-Behalf-Of (OBO) flow.
```

Comment:
I think we want to bump 3.10 to 3.11 - it isn't required for NAA to work, but it is the most recent MSAL.js release / their 'current' version.